src/Helper/HeadScript.php

```diff
@@ -33,6 +33,7 @@ class HeadScript extends Placeholder\Container\AbstractStandalone
      */
     const FILE   = 'FILE';
     const SCRIPT = 'SCRIPT';
+    const DEFAULT_SCRIPT_TYPE = 'text/javascript';
```

Comment:
Let's declare `private` visibility on this one, as it's internal; that way we're not also changing the external API for a bugfix.

Reply:
private would require PHP 7.1, see comments below. IMO changing PHP Version would be a BC Break